test/framework/util.go

```diff
@@ -103,24 +103,25 @@ func WaitFor(namespace, display string, timeout time.Duration, condition func()
 	}
 }
 
-// GetScenarioName retrieves the name of the scenario
-func GetScenarioName(s interface{}) string {
-	if scenario, ok := s.(*gherkin.Scenario); ok {
-		return scenario.Name
-	}
-	return s.(*gherkin.ScenarioOutline).Name
-}
-
-func GetExamplesNames(outline *gherkin.ScenarioOutline) (names []string) {
-	for _ , examples := range outline.Examples {
-		examplesName := examples.Name
-		if len(examples.Name) == 0 {
-			examplesName = "Unnamed Examples"
-		}
-		names = append(names, examplesName)
-	}
-	return
-}
+// // GetScenarioName retrieves the name of the scenario
```

Comment:
What is usually your approach about commented blocks like this? I normally remove the commented blocks as we already have the source repository to track changes.

Reply:
that I should delete indeed

Reply:
deleted